task/git-clone-community/0.1/patch.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- op: replace
+  path: /metadata/name
+  value: git-clone-community
+
+- op: replace
+  path: /spec/steps/0/image
+  value: gcr.io/tekton-releases/github.com/tektoncd/pipeline/cmd/git-init:v0.40.2
```

Comment:
We would need a mechanism to keep this image up to date as well as a mechanism to keep `task/git-clone-community/0.1/git-clone.yaml` in sync with with the "non-community" version

But:
* What rule prohibits using community images for the redhat deployment of Konflux? Can we have just one git-clone task?
* If not, can the community pipeline use the [community git-clone task](https://hub.tekton.dev/tekton/task/git-clone)?

Reply:
> I assumed that we need the tasks to be provided from our bundle to for passing enterprise-contract.

Hmm, I think that will soon put the build team in an impossible situation, having to maintain all the tasks for the konflux-ci community

Reply:
How hard will it be to get rid of the git-clone-community task if we do create it?

I would want to get rid of it ASAP and get the openshift-pipelines team to release the git-init image to registry.access.redhat.com. But if someone starts using community pipelines by then, it could be really difficult.